### DISCUSSION TITLE
Frontend Better Practices

### DISCUSSION THREAD

**Discussion by rahulrana95** [SigNoz/signoz#205] [Ideas]
https://docs.google.com/document/d/1TQr2q3fpUtlDl2mNjvLj-dae_3o_d0JUG5xOBSVG_qk/edit?usp=sharing

**Comment by ankitnayan** [comment]
I liked the point @rahulrana95 made that though multiple devs are contributing to the project, it should seem like one and following coding practices and the same design principles all through the project is very important. I would love to see this in the above shared docs. Please takeout some time to define some groundwork and principles and inputs will be highly appreciated @rahulrana95 @nidhi-tandon @jyash97

**Comment by palashgupta98** [comment]
@rahulrana95 just updated the docs with Cypress for end to end testing
